docs/file-format.rst

```diff
@@ -535,3 +535,36 @@ Path                               Type
 /trees/indexes/insertion_order     H5T_STD_U32LE
 /trees/indexes/removal_order       H5T_STD_U32LE
 ==============================     ==============
+
+***************
+Legacy Versions
+***************
+
+Tree sequence files written by older versions of msprime are not readable by
+newer versions of msprime. For major releases of msprime, :command:`msp
+upgrade` will convert older tree sequence files to the latest version.
+
+However many changes to the tree sequence format are not part of major
+releases. The table below gives these versions (contained in the root group
+attribute, ``format_version`` as a pair ``(major, minor)``).
+
+================    =================
+Version             Commit Short Hash
+================    =================
+HDF5_FORMAT_V9.0    e504abd
```

Comment:
We can probably just cut this down to 9.0 etc. here. This is probably more readable I think.